varnish/check.py

```diff
@@ -112,25 +129,25 @@ def _get_version_info(self, varnishstat_path):
 
         # Assumptions regarding varnish's version
         use_xml = True
-        version = 3
+        version = [3, 0, 0]
 
-        m1 = re.search(r"varnish-(\d+)", output, re.MULTILINE)
+        m1 = re.search(r"varnish-(\d+\.\d+\.\d+)", output, re.MULTILINE)
```

Comment:
let's get rid of these regexp in favor of `distutils`, see https://github.com/DataDog/integrations-core/blob/817489ebc3a7b4b87399820a6c43bc5915f11891/zk/check.py#L205 for an example